src/renderer/components/MainPage.jsx

```diff
@@ -114,16 +115,19 @@ export default class MainPage extends React.Component {
 
   getTabStatus() {
     // TODO: should try to make this a bit safer in case we get into a weird situation
```

Comment:
Still need this TODO?

Reply:
it could always be safer.... but no, no need for it anymore :) thanks for catching that